src/Cache/_ScsDataClient.php

```diff
@@ -175,6 +177,14 @@ private function ttlToMillis(?int $ttl = null): int
         return $ttl * 1000;
     }
 
+    private function returnCollectionTtl($ttl): CollectionTtl
```

Comment:
can we annotate tye type of the argument here?  I believe it would be `?CollectionTtl`?